src/rgw/rgw_lc.cc

```diff
@@ -855,7 +879,8 @@ int RGWLC::handle_multipart_expiration(rgw::sal::Bucket* target,
         return ret;
       }
 
-      ret = mpu->abort(this, cct, null_yield);
+      bool log_op = !zonegroup_lc_check(wk->get_lc(), driver->get_zone());
+      ret = mpu->abort(this, cct, null_yield, log_op);
```

Comment:
multipart uploads wouldn't get replicated until CompleteMultipart - is there a reason we'd ever need to log aborts?

Reply:
The reason why I added the flag check here was that I noticed the multipart operations are actually added to the bilog. Please see below for a snippet of the bilog output of my test. This was done in Ceph main without changes in this PR. I'm not 100% sure if they are there for a purpose. 
Why they are in the replication logs? Was it because the code was not properly cleaned up? Is it safe to remove them without any impact on all the sync modules?
```
    {
        "op_id": "42#00000000002.1573176.6",
        "op_tag": "_8j_0u234dij9u-hpemWsCzqi93sFS0g",
        "op": "write",
        "object": "_multipart_file_17m.2~08kY-1D9wwP5Lqgu4DZlTOFjVsctKbC.1",
        "instance": "",
        "state": "complete",
        "index_ver": 2,
        "timestamp": "2023-12-07T05:39:00.239036988Z",
        "ver": {
            "pool": 8,
            "epoch": 0
        },
        "bilog_flags": 0,
        "versioned": false,
        "owner": "",
        "owner_display_name": "",
        "zones_trace": [
            {
                "entry": "baec2a9c-611c-49b9-a3d0-917ceb7a99ff:bucket1:baec2a9c-611c-49b9-a3d0-917ceb7a99ff.12871.1"
            }
        ]
    },

```

Reply:
agreed. we don't need to add these parts into bilog.

Reply:
my past self agrees with this too: https://tracker.ceph.com/issues/21800 i added the multisite-backlog tag there

Reply:
Cool. Then I'll remove the lc changes here and leave it to be handled in the  https://tracker.ceph.com/issues/21800. 
And also since since I'm here, I'll take that issue (https://tracker.ceph.com/issues/21800) as well.

Reply:
the multipart related lc changes in rgw_lc.cc have been removed.